packages/expo-permissions/src/Permissions.ts

```diff
@@ -63,7 +78,16 @@ async function _handleSinglePermissionRequestIOSAsync(
       canAskAgain: true,
     };
   }
-  return await handlePermission(type);
+  try {
+    return await handlePermission(type);
```

Comment:
One thing I haven't been able to figure out is handling the android part, since android permission can ask multiple permissions at once, so extracting unknown one is quite not straight forward

Reply:
I think what you did is fine for now 😉 👍